### ISSUE TITLE
Version 3.3.0 broke python 3.8 support with invalid typehinting

### ISSUE THREAD

**Issue by SirStigs** [kislyuk/watchtower#205]
```
venv/lib/python3.8/site-packages/watchtower/__init__.py:216: in CloudWatchLogHandler
    log_group_tags: dict[str, str] = {},
E   TypeError: 'type' object is not subscriptable
```

Looks like missing a `from __future__ import annotations` from the top (the __init__.py used to use from Typing import....)

**Comment by faraggi** [comment]
FWIW, I'm also having this issue now. I temporarily pinned the version to 3.2.0 until a fix is available.

**Comment by ckz8780** [comment]
Same issue here, but weirdly I tried pinning to 3.2.0 yesterday and it didn't work. Today I pinned boto to 1.34.117 (my last known working version) and now my deploy is working again but watchtower is still 3.3.0 in my lockfile. 

Python 3.8.12 fwiw

**Comment by kislyuk** [comment]
Fixed in v3.3.1. Sorry about the hassle.
